_includes/guide/coverage_resource.html

```diff
@@ -1,3 +1,3 @@
 <p>
-The <a href="https://www.hl7.org/fhir/coverage.html" target="_blank" rel="noopener" class="in-text__link"> Coverage</a> resource type indicates the beneficiary’s enrollment record. It includes information on a beneficiary’s Part A, Part B, Part C, and Part D coverage, as well as markers for End-stage Renal Disease (ESRD) and Old Age and Survivors Insurance (OASI).
+The <a href="https://www.hl7.org/fhir/coverage.html" target="_blank" rel="noopener" class="in-text__link"> Coverage</a> The Coverage resource type provides information about beneficiaries’ insurance coverage, including information about dual coverage.
```

Comment:
Duplicate wording, this should be `The <a href="https://www.hl7.org/fhir/coverage.html" target="_blank" rel="noopener" class="in-text__link"> Coverage</a> resource type provides information about beneficiaries’ insurance coverage, including information about dual coverage.`